mdoc/Mono.Documentation/Updater/Statistics/StatisticsStorage.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Mono.Documentation.Updater.Statistics
+{
+    public class StatisticsStorage
```

Comment:
Please add some triple-slash documentation with at least a summary description of this class' intended use